lib/src/util/posix/pid_file.cc

```diff
@@ -210,11 +210,9 @@ bool PIDFile::canLockFile(int fd, int lock_type) {
     fl.l_start = 0;
     fl.l_len = 0;
 
-    if (fcntl(fd, F_GETLK, &fl) == -1) {
+    if (fcntl(fd, F_GETLK, &fl) == -1)
         // Failure
-        std::string msg { "errno=" + std::to_string(errno) };
-        throw PIDFile::Error { msg };
-    }
+        throw PIDFile::Error { lth_loc::format("errno={1}", errno) };
```

Comment:
:( I'm surprised our linter lets this pass. Oh well, maybe it catches if you have two indented lines.